### PR TITLE
add linters for clean code and correct few formatting

### PR DESCRIPTION
**Pull request by GwonsooLee**
Issue: #12 

- Add few tools for clean code and apply to all of the source code
- Change root cmd in order to manage command by group
- Add `logrus` for logging ( -v will be applied for all commands )
- Add contributing guide

## Please check the rule for goimport
- packages in the same repository should come at last and all section need to be separated with new line.
```
import (
    <internal package>

    <packages from other repository>

    <packages in the same repository>
)
```

## Make sure that all users should run `make linters` before creating PR
```
❯ make linters
Running linters...
RUN hack/boilerplate.sh
PASSED hack/boilerplate.sh in 0s
RUN hack/gofmt.sh
PASSED hack/gofmt.sh in 1s
RUN hack/pedantic-imports.sh
PASSED hack/pedantic-imports.sh in 0s
RUN hack/golangci-lint.sh
PASSED hack/golangci-lint.sh in 1s
```

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## Process
+1. Set up local development environment.
+2. Change something...
+3. Run test.
+4. Create PR.
+
+### 1. Set up local development environment
+- You have to install...
+    - golang ( >= 1.14 )
+- You have to have Amazon EKS Cluster with kubernetes version 1.16 or higher
```
Comment (asbubam) [inline CONTRIBUTING.md:13]: * klocust can run with any K8s clusters include EKS. :)

Comment (GwonsooLee) [reply CONTRIBUTING.md]: fixed :) 
Thanks ~ 👍

File: README.md [hunk 2]
```diff
@@ -69,6 +69,8 @@ $ klocust apply hello -cf hello-klocust.yaml -lf hello-locustfile.yaml
 $ klocust delete hello
 ```
 
+[build from source]: ./docs/build_from_source.md
 
 
-[build from source]: ./docs/build_from_source.md
+## Contribution Guide
```
Comment (asbubam) [inline README.md:75]: Could you move `Contribution Guide` upper to `build from source` page link? (Sorry, I don't know right name of that).

File: cmd/klocust/version.go [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+	"context"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	"github.com/DevopsArtFactory/klocust/internal/version"
+)
+
+// Get klocust version
+func NewVersionCmd() *cobra.Command {
```
Comment (asbubam) [inline cmd/klocust/version.go:29]: I`ve really want it.

File: cmd/klocust/root.go [hunk 4]
```diff
@@ -1,30 +1,126 @@
...
-	ExactArgs(0, doRoot)
+			return nil
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return cmd.Help()
+		},
+	}
+
+	//Group by commands
+	groups := templates.CommandGroups{
```
Comment (asbubam) [inline cmd/klocust/root.go:68]: Oh, I didn't know grouping command! Thanks.

**Review by asbubam — COMMENTED** [review]
Thank you for contribution!
I didn't know that about import rules exactly. Thanks.

File: README.md (hunk 2 above)
Comment (GwonsooLee) [reply README.md:75]: Oh Okay! You already made this. I will delete CONTRIBUTING.md and link build_from_source.md to it

Comment (GwonsooLee) [reply README.md:75]: No... I did something stupid... Never mind.

I will change the listing order

**Review by asbubam — APPROVED** [review]
👍

**Pull request merged**